cypher/cypher-docs/src/docs/dev/ql/administration/security/admin-role-introduction.asciidoc

```diff
@@ -0,0 +1,116 @@
+The `DENY` command allows an administrator to deny a privilege to a role in order to prevent access to an entity.
+The `REVOKE` command allows an administrator to remove a previously granted or denied privilege.
+The syntax is:
+
+.Privilege command syntax
+[options="header", width="100%", cols="3a,2"]
+|===
+| Command | Description
+
+| [source, cypher]
+GRANT database-privilege ON DATABASE dbname TO role
+| Grant a privilege to a role
+
+| [source, cypher]
+DENY database-privilege ON DATABASE dbname TO role
+| Deny a privilege to a role
+
+| [source, cypher]
+REVOKE GRANT database-privilege ON DATABASE dbname FROM role
+| Revoke a granted privilege from a role
+
+| [source, cypher]
+REVOKE DENY database-privilege ON DATABASE dbname FROM role
+| Revoke a denied privilege from a role
+
+| [source, cypher]
+REVOKE database-privilege ON DATABASE dbname FROM role
+| Revoke a granted or denied privilege from a role
+|===
+
+[NOTE]
+These commands are very similar to the <<administration-security-subgraph-introduction, graph-privileges>> except for the use of the term `DATABASE`, no `entity` and and the set of available _database-privileges_ differs from the _graph-privileges_.
+
+Where the components are:
+
+* _database-privilege_
+** `ACCESS`
++
+allows access for a specific database/graph
++
+** `START`
++
+allows the specified database to be started
++
+** `STOP`
++
+allows the specified database to be stopped
++
+** `CREATE INDEX`
++
+allows indexes to be created on the specified database.
++
+** `DROP INDEX`
++
+allows indexes to be deleted on the specified database.
++
+** `INDEX MANAGEMENT`
++
+allows indexes to be created and deleted on the specified database.
++
+** `CREATE CONSTRAINT`
++
+allows constraints to be created on the specified database.
++
+** `DROP CONSTRAINT`
++
+allows constraints to be deleted on the specified database.
++
+** `CONSTRAINT MANAGEMENT`
++
+allows constraints to be created and deleted on the specified database.
++
+** `CREATE NEW [NODE] LABEL`
++
+allows labels to be created so that future nodes can be assigned them.
++
+** `CREATE NEW [RELATIONSHIP] TYPE`
++
+allows relationship types to be created so that future relationships can be created with these types.
++
+** `CREATE NEW [PROPERTY] NAME`
++
+allows property names to be created so that nodes and relationships can have properties with these names assigned.
++
+** `NAME MANAGEMENT`
++
+allows all of the name management capabilities: node labels, relationship types and property names.
++
+** `ALL [[DATABASE] PRIVILEGES]`
++
+allows all of the above privileges to be enabled for the specified database.
++
+* _dbname_
+** The database to associate the privilege with.
+   Note that if you delete a database and create a new one with the same name, the new one will NOT have any of the privileges specifically assigned to the deleted graph.
```

Comment:
`privileges specifically assigned to the deleted graph.` should be database not graph?